src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -69,7 +69,14 @@ public class DocumentBuilder : IDisposable
                 Logger.LogVerbose($"\t{processor.Name} with build steps ({string.Join(", ", from bs in processor.BuildSteps orderby bs.BuildOrder select bs.Name)})");
             }
             _postProcessors = GetPostProcessor(postProcessorNames);
-            _intermediateFolder = intermediateFolder;
+            if (intermediateFolder != null)
+            {
+                if (intermediateFolder.Length == 0)
+                {
+                    _intermediateFolder = Path.GetFullPath(".");
```

Comment:
Path.GetFullPath("") will throw expection.

---
In reply to: [93723348](https://github.com/dotnet/docfx/pull/1039#discussion_r93723348) [](ancestors = 93723348)